alphabase/constants/aa.py

```diff
@@ -53,13 +58,11 @@ def reset_AA_df():
     AA_DF = pd.DataFrame()
     AA_DF["aa"] = [chr(aa) for aa in range(len(AA_ASCII_MASS))]
     AA_DF["formula"] = [""] * len(AA_ASCII_MASS)
-    aa_idxes = []
-    formulas = []
-    for aa, formula in AA_Formula.items():
-        aa_idxes.append(ord(aa))
-        formulas.append(formula)
-    AA_DF.loc[aa_idxes, "formula"] = formulas
+    AA_DF["smiles"] = [""] * len(AA_ASCII_MASS)
```

Comment:
could you introduce `num_rows =  len(AA_ASCII_MASS)` and use it throught this method?